app/assets/javascripts/admin/controllers/modals/admin-install-theme.js.es6

```diff
@@ -0,0 +1,278 @@
+    meta_url: "https://meta.discourse.org/t/discourse-vincent-theme/76662"
+  },
+  {
+    name: "Alternative Logos",
+    value: "https://github.com/hnb-ku/discourse-alt-logo",
+    meta_url:
+      "https://meta.discourse.org/t/alternative-logo-for-dark-themes/88502",
+    component: true
+  },
+  {
+    name: "Brand Header Theme Component",
+    value: "https://github.com/discourse/discourse-brand-header",
+    meta_url: "https://meta.discourse.org/t/brand-header-theme-component/77977",
+    component: true
+  },
+  {
+    name: "Custom Header Links",
+    value: "https://github.com/hnb-ku/discourse-custom-header-links",
+    preview:
+      "https://theme-creator.discourse.org/theme/Johani/custom-header-links",
+    meta_url: "https://meta.discourse.org/t/custom-header-links/90588",
+    component: true
+  },
+  {
+    name: "Category Banners",
+    value: "https://github.com/awesomerobot/discourse-category-banners",
+    preview:
+      "https://theme-creator.discourse.org/theme/awesomerobot/discourse-category-banners",
+    meta_url: "https://meta.discourse.org/t/discourse-category-banners/86241",
+    component: true
+  },
+  {
+    name: "Hamburger Theme Selector",
+    value: "https://github.com/discourse/discourse-hamburger-theme-selector",
+    meta_url: "https://meta.discourse.org/t/hamburger-theme-selector/61210",
+    component: true
+  },
+  {
+    name: "Header submenus",
+    value: "https://github.com/hnb-ku/discourse-header-submenus",
+    preview: "https://theme-creator.discourse.org/theme/Johani/header-submenus",
+    meta_url: "https://meta.discourse.org/t/header-submenus/94584",
+    component: true
+  }
+];
+
+const POPULAR_COMPONENTS = [];
+
+export default Ember.Controller.extend(ModalFunctionality, {
+  popular: Ember.computed.equal("selection", "popular"),
+  local: Ember.computed.equal("selection", "local"),
+  remote: Ember.computed.equal("selection", "remote"),
+  create: Ember.computed.equal("selection", "create"),
+  selection: "popular",
+  adminCustomizeThemes: Ember.inject.controller(),
+  loading: false,
+  keyGenUrl: "/admin/themes/generate_key_pair",
+  importUrl: "/admin/themes/import",
+  checkPrivate: Ember.computed.match("uploadUrl", /^git/),
+  localFile: null,
+  uploadUrl: null,
+  urlPlaceholder: "https://github.com/discourse/sample_theme",
+  advancedVisible: false,
+  themesController: Ember.inject.controller("adminCustomizeThemes"),
+  createTypes: [
+    { name: I18n.t("admin.customize.theme.theme"), value: THEMES },
+    { name: I18n.t("admin.customize.theme.component"), value: COMPONENTS }
+  ],
+
+  @computed("themesController.installedThemes")
+  themes(installedThemes) {
+    return POPULAR_THEMES.map(t => {
+      if (installedThemes.includes(t.name)) {
+        Ember.set(t, "installed", true);
+      }
+      return t;
+    });
+  },
+
+  @computed("themesController.installedThemes")
+  popularComponents(installedThemes) {
```

Comment:
Is this used? A search for `popularComponents` comes up with nothing and `POPULAR_COMPONENTS` is an empty array.